server/pyproject.toml

```
@@ -232,7 +232,7 @@ ignore_errors = true
 requires = [
     "poetry-core>=1.0.0",
     "setuptools",
-    "maturin==0.15.1",
+    "maturin==1.1.0",
     "maturin[patchelf]==0.15.1; platform_system=='Linux'",
```

This line need to be updated
```suggestion
    "maturin[patchelf]==1.1.0; platform_system=='Linux'",
```